src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6880,6 +6883,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_CollectionExpressionKeyValuePairNotSupported" xml:space="preserve">
     <value>Collection expression type '{0}' does not support key-value pair elements.</value>
   </data>
+  <data name="ERR_CollectionArgumentsMustBeFirst" xml:space="preserve">
+    <value>Collection arguments must be the first element.</value>
+  </data>
```

Comment:
Grr * 2.

Note i think this should be called "collection argument element" or just "'with' element. Saying the *arguments* (plural) must be the first element (singular) reads oddly to me.